website/src/pages/docs/guides/react-vue.mdx

```diff
@@ -45,25 +45,9 @@ All the below code examples are available in our [repository `examples` folder](
 
 For most GraphQL clients and frameworks (React, Vue), install the following packages:
 
-**For Yarn:**
-
-```bash
-yarn add graphql
-yarn add -D typescript ts-node @graphql-codegen/cli @graphql-codegen/client-preset @parcel/watcher
-```
-
-**For npm:**
-
-```bash
-npm i -S graphql
-npm i -D typescript ts-node @graphql-codegen/cli @graphql-codegen/client-preset @parcel/watcher
-```
-
-**For pnpm:**
-
-```bash
-pnpm i graphql @graphql-typed-document-node/core
-pnpm i -D typescript ts-node @graphql-codegen/cli @graphql-codegen/client-preset @parcel/watcher
+```sh npm2yarn
+npm install graphql
+yarn install --dev typescript @graphql-codegen/cli @parcel/watcher
```

Comment:
The `yarn install ...` statement seems to make both `pnpm` and `bun` use `yarn` command. I think this could be a copy/pasta issue?

<img width="890" alt="Screenshot 2024-07-29 at 8 57 03 PM" src="https://github.com/user-attachments/assets/56197f6e-9311-41e3-843e-821bedb474c0">

Reply:
done!